modmesh/pilot/_burgers1d.py

```diff
@@ -0,0 +1,260 @@
+import sys
+from dataclasses import dataclass
+
+import numpy as np
+
+import matplotlib
+import matplotlib.pyplot
+from matplotlib.backends.backend_qtagg import FigureCanvas
+from matplotlib.figure import Figure
+from matplotlib.pyplot import setp
+
+from PySide6.QtCore import QTimer, Slot, Qt
+from PySide6.QtWidgets import QDockWidget, QWidget
+
+from .. import core as mcore
+from ._gui_common import PilotFeature
+from ._euler1d import SolverConfig, PlotConfig, PlotArea, ConfigWindow
+
+
+@dataclass
+class QuantityLine(object):
```

Comment:
You should use some docstrings.  Please do it in a follow-up PR.

Reply:
This is common code shared between multiple physical problems.  It should be factored out.